internal/provider/team_data_source_test.go

```diff
@@ -1,23 +1,33 @@
 package provider_test
 
-//func TestAccTeamDataSource(t *testing.T) {
-//	resource.Test(t, resource.TestCase{
-//		PreCheck:                 func() { testAccPreCheck(t) },
-//		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
-//		Steps: []resource.TestStep{
-//			// Read testing
-//			{
-//				Config: testAccTeamDataSourceConfig,
-//				Check: resource.ComposeAggregateTestCheckFunc(
-//					resource.TestCheckResourceAttr("data.cortex_team.platform_engineering", "tag", "platform_engineering"),
-//				),
-//			},
-//		},
-//	})
-//}
-//
-//const testAccTeamDataSourceConfig = `
-//data "cortex_team" "engineering" {
-//  tag = "platform_engineering"
-//}
-//`
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccTeamDataSource(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Read testing
+			{
+				Config: testAccTeamDataSourceConfig,
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("data.cortex_team.engineering", "tag", "platform_engineering"),
+					resource.TestCheckResourceAttrSet("data.cortex_team.engineering", "members.0.name"),
+					resource.TestCheckResourceAttrSet("data.cortex_team.engineering", "members.0.email"),
+					resource.TestCheckResourceAttrSet("data.cortex_team.engineering", "slack_channels.0.name"),
+					resource.TestCheckResourceAttrSet("data.cortex_team.engineering", "slack_channels.0.notifications_enabled"),
```

Comment:
Can we create a stub resource so that we can make more meaningful assertions here? The first one is great, but the latter four could simply be set to nonsense data and this test wouldn't catch that.